02-microarray/pathway-analysis_microarray_03_gsea.Rmd

```diff
@@ -275,7 +282,8 @@ Let's see a preview of `dge_mapped_df`.
 head(dge_mapped_df)
 ```
 
-Now let's check to see if we have any Entrez IDs that mapped to multiple Ensembl IDs.
+We do not want duplicated gene identifiers for the GSEA steps later as the `GSEA()` function will throw a warning that having duplicate gene names may produce unexpected results.
```

Comment:
I attempted to address this comment in the last commit via some restructuring and rewording. Please let me know if what I pushed in the latest commits are suffice (I was not sure if I should go more in depth about the warning message since users will not actually see it here).